addon/components/new-osf-navbar/template.hbs

```diff
@@ -51,7 +51,7 @@
                             {{/nav.item}}
                         {{else}}
                             {{#nav.item}}
-                                <a href="{{navLink.href}}" onclick={{action 'click' 'link' (concat 'Navbar - ' navlink.name)}}>
+                                <a href="{{serviceSubmitLink}}" onclick={{action 'click' 'link' (concat 'Navbar - ' navlink.name)}}>
```

Comment:
This looks like it changes the URL for *every* link in the navbar, not just "Add Preprint". It might be better to do this in the `build-secondary-nav-links` helper, and leave `new-osf-navbar` unchanged.